docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc

```diff
@@ -2,13 +2,111 @@
 = Adding commands to a devfile
 
 [role="_abstract"]
-A devfile allows to specify commands to be available for execution in a workspace. Every command can contain a subset of actions. These subsets are related to specific components and are executed in the component containers. See the following list for command properties in a devfile:
+A devfile allows you to specify commands to be available for execution in a workspace. Every command can contain a subset of actions. These subsets are related to specific components and are executed in the component containers. See the following tables for command properties in a devfile:
 
-[horizontal]
-Key:: commandObject
-Type:: commandObject
-Required:: no
-Description:: command to be executed in an existing component container
+.commandObject
+[cols="1,1,1,1"]
+|===
+|Key |Type| Required| Description
+
+|exec
+|execObject
+|no
+|The exec command to run.
+
+|composite
+|compositeObject
+|no
+|The composite command to run.
```

Comment:
Shouldn't this explain what the composite command is for, and in what situation would user use this?
Or if we are already doing this somewhere else, this should probably link to it. 


(the same for `exec` command)

Reply:
+1 The composite command description should probably at least say it is chaining multiple commands together.

Reply:
I added that composite commands chain multiple commands together. We can improve the description in a later PR.